.github/workflows/complete.yml

```diff
@@ -17,10 +17,16 @@ jobs:
       - uses: actions/checkout@v2
         with:
           submodules: recursive
+      - uses: actions/checkout@v2
+      - uses: docker/setup-qemu-action@v1
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
       - uses: GoogleCloudPlatform/github-actions/setup-gcloud@master
         with:
           version: '290.0.1'
           export_default_credentials: true
+          project_id: ${{ secrets.GCP_PROJECT_ID }}
```

Comment:
Which GCP_PROJECT_ID is this using, and dont we need builds for PRs?

Reply:
this is `kf-feast`. For PRs we still use hosted runner. At least I don't see a lot of alternatives if we want to continue to use prow